packages/office-ui-fabric-react/src/components/Button/ButtonBase.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css,
+  assign,
+  getId,
+  getNativeProps,
+  buttonProperties,
+  anchorProperties
+} from '../../Utilities';
+import { IButtonProps, IButton, ButtonType } from './Button.Props';
+
+export interface IButtonState {
+  labelId?: string;
+  descriptionId?: string;
+  ariaDescriptionId?: string;
+}
+
+export abstract class ButtonBase extends BaseComponent<IButtonProps, IButtonState> implements IButton {
+  public static defaultProps: IButtonProps = {
+    buttonType: ButtonType.default
+  };
+  private _buttonElement: HTMLButtonElement;
+
+  protected getRootClassName() { return ''; }
```

Comment:
Not sure if this is how I should do this @dzearing, but at least it's working as intended :)